pkgs/top-level/all-packages.nix

```diff
@@ -23207,6 +23207,8 @@ with pkgs;
   SDL2 = callPackage ../development/libraries/SDL2 {
     inherit (darwin.apple_sdk.frameworks) AudioUnit Cocoa CoreAudio CoreServices ForceFeedback OpenGL;
   };
+  # SDL2_2_24 pinned for pygame
+  SDL2_2_24 = callPackage ../development/libraries/SDL2/2_24.nix { };
```

Comment:
Instead of accumulating new versions in top-level we should do this in the package if it is really only needed for one package.

Reply:
Problem is, then another application needs library at that version and isn't available. Which has happened before.
I am waiting for staging builds to realize what else will need to be pinned to 2.24.

Reply:
I don't think this is a case that is helpful in keeping hidden. Because SDL2 is used by several thousands of packages. The likely hood of one application not playing nice with latest version is high.